modules/Auth/tests/auth_tests.cpp

```diff
@@ -15,6 +15,20 @@ using ::testing::MockFunction;
 using ::testing::StrictMock;
 using ::testing::_;
 
+namespace types
+{
+    namespace authorization {
+        std::ostream & operator<<(std::ostream & os, TokenValidationStatus const & status) {
+            return os << types::authorization::token_validation_status_to_string(status);
+        }
+    }
+    namespace reservation {
+        inline std::ostream& operator<<(std::ostream& os, const ReservationResult& reservation_result) {
+            return os << types::reservation::reservation_result_to_string(reservation_result);
+        }
+    }
+}
+
```

Comment:
This probably shouldn't be necessary anymore with current ev-cli (see EVerest/everest-utils#84)

Reply:
Awesome! I will drop this commit :) I actually put this comment in the commit message:
```
This is quite controversial commit. Since the operators are generated
outside of the namespace, GTest does not find them.
Either this commit is kept as it is or the generators should generate
the operators inside of the namespace where the enums are defined. This
could have some influence on the rest of the project, so it's an open
discussion point
```